core/drivers/crypto/se050/core/ecc.c

```diff
@@ -18,6 +20,19 @@
 #include <utee_defines.h>
 #include <util.h>
 
+static const struct crypto_ecc_keypair_ops *pair_ops;
+static const struct crypto_ecc_public_ops *pub_ops;
+
+static bool oefid_key_supported(size_t bits)
```

Comment:
commit "crypto: drivers: se050: ecc: fallback to software operations" needs `__unused` here and for `oefid_algo_supported()`.